web-components/src/components/favorite/Favorite.ts

```diff
@@ -17,7 +18,7 @@ import { classMap } from "lit-html/directives/class-map";
 import { ifDefined } from "lit-html/directives/if-defined";
 
 export namespace Favorite {
-  @customElement("md-favorite")
+  @customElementWithCheck("md-favorite")
```

Comment:
Lena please add CONTRIBUTING doc. You also need to add an endpoint in webpack file

Reply:
Done;